CAVaccineInventory/site
# fix: re-add self to coordinator

At some point I fell off the coordinator list. Re-adding.

Link to Deploy Preview: https://deploy-preview-225--vaccinateca.netlify.app/

---

### Manual Testing (QA)

_not performed since this is a minor content change_

Open the deploy preview and manually perform the following tests with the browser's developer console open. Fix or log any unexpected errors you see in the console. Check off the following manual tests as you go through them. Make sure to resize the screen and check for poorly positioned or spaced items at mobile and tablet sizes (60%+ of our audience is on mobile devices).

#### Other Pages

#### Did you remember to:
